dynamics/rolldowndata/rolldown/smooth_rolldown_data.py

```diff
@@ -0,0 +1,72 @@
+from csv import reader, writer
+from scipy.signal import savgol_filter
+import matplotlib.pyplot as plt
+import argparse
+import sys
+import os.path
+sys.path.append(os.path.dirname(sys.path[0]))
```

Comment:
So this was initially all wrapped in a function, which may be helpful if we decide to test, but it felt like a script. Any thoughts?

Reply:
Makes sense to me as a script, although I'm not the most aware of how this might affect our setup further down the line but I'm fine with it

Reply:
We can merge as a script and fix as needed later. As we figure out how all our code is interacting with itself, we'll probably need to make some adjustments anyways.